.github/workflows/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "hourly"
+    target-branch: "cleaning/dependabot"
+    commit-message:
+      prefix: "chore:"
```

```suggestion
      prefix: "chore"
```

The docs make it sound like the colon is added for us. No need to specify it.